transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+sources:
+  - name: building_footprints
+    database: raw_dev
+    schema: building_footprints
+    tables:
+      - name: california_building_footprints
+  - name: tiger_2022
+    database: raw_dev
+    schema: tiger_2022
+    tables:
+      - name: blocks
+      - name: places
+
+models:
+  - name: geo_reference__blocks_joined
```

Comment:
Thinking about this a bit more: I still think this name needs some work: it doesn't have anything about building footprints, in it, and geo_reference seems not particularly descriptive. Maybe something along the lines of  `geo_reference__building_footprints_with_blocks`? Would be great to get @AeriShan-ODI's opinion here as well

Reply:
Accepting this suggestion for now unless there's further discussion and suggestions to consider.